docs/package-lock.json

```diff
@@ -2797,11 +2797,11 @@
 			"dev": true
 		},
 		"@vue/composition-api": {
-			"version": "1.2.1",
-			"resolved": "https://registry.npmjs.org/@vue/composition-api/-/composition-api-1.2.1.tgz",
-			"integrity": "sha512-Gea21Vyl2PVDLc6Fq1rr5poI2qU+1WjGze6oykWDSiGEII72n11XdZOSqF6BA53CXZZVs27cz3S3mfJM0a59Vw==",
+			"version": "1.4.3",
```

Comment:
Hey, @jakobw 👋  Even if it's minor, I'm not sure whether this version bump might cause any issues. Do you think we can merge?

Reply:
Yup, looks safe. I also tried these two out locally to see if anything explodes and it seems it's all fine!